rtublen/Studentportalen
# US0830-Add-more-information

pages/profiles/FredrikAndersson.cshtml

# How to verify

1. Start application
2. go to /profiles/fredrikandersson
3. Verify that you can visit the page
4. Verify that the content is ok
